README.md

```diff
@@ -171,6 +171,12 @@ or [GitLab examples](https://gitlab.com/UncleRus/esp-idf-lib/tree/master/example
 | **sht4x**                | Driver for Sensirion SHT40/SHT41/SHT45 digital temperature and humidity sensor   | BSD-3   | `esp32`, `esp8266`, `esp32s2`, `esp32c3` | Yes
 | **si7021**               | Driver for Si7013/Si7020/Si7021/HTU2xD/SHT2x and compatible temperature and humidity sensors | BSD-3   | `esp32`, `esp32c3`, `esp8266`, `esp32s2`, `esp32c3` | Yes
 
+### Inertial measurement unit (IMUs)
```

Comment:
Please do not edit README.md manually, use [this](https://github.com/UncleRus/esp-idf-lib/blob/master/CONTRIBUTING.md#updating-readmemd).
Also add yourself to the contributors list in `devtools/README.md.erb`